app/models/pagination.rb

```diff
@@ -0,0 +1,13 @@
+class Pagination < ApplicationStruct
+  attribute :current_page, Types::Params::Integer.optional.default(1)
+  attribute? :limit_value, Types::Params::Integer.default(50)
```

Comment:
Also think perhaps a constant could make this number clearer - what is a "limit_value" to someone looking at this for the first time. I don't think its completely obvious.

`attribute? :limit_value, Types::Params::Integer.default(APPLICATIONS_PER_PAGE)`

May clear this up a little :)

Reply:
👍